ethereum/core/src/main/java/org/hyperledger/besu/ethereum/mainnet/ProtocolScheduleBuilder.java

```diff
@@ -48,59 +48,12 @@ public class ProtocolScheduleBuilder {
   private final PrivacyParameters privacyParameters;
   private final boolean isRevertReasonEnabled;
   private final EvmConfiguration evmConfiguration;
-  private final MiningParameters miningParameters;
   private final BadBlockManager badBlockManager;
   private final boolean isParallelTxProcessingEnabled;
   private final MetricsSystem metricsSystem;
+  private final MiningParameters miningParameters;
 
   public ProtocolScheduleBuilder(
-      final GenesisConfigOptions config,
-      final BigInteger defaultChainId,
-      final ProtocolSpecAdapters protocolSpecAdapters,
-      final PrivacyParameters privacyParameters,
-      final boolean isRevertReasonEnabled,
-      final EvmConfiguration evmConfiguration,
-      final MiningParameters miningParameters,
-      final BadBlockManager badBlockManager,
-      final boolean isParallelTxProcessingEnabled,
-      final MetricsSystem metricsSystem) {
-    this(
-        config,
-        Optional.of(defaultChainId),
-        protocolSpecAdapters,
-        privacyParameters,
-        isRevertReasonEnabled,
-        evmConfiguration,
-        miningParameters,
-        badBlockManager,
-        isParallelTxProcessingEnabled,
-        metricsSystem);
-  }
-
-  public ProtocolScheduleBuilder(
-      final GenesisConfigOptions config,
-      final ProtocolSpecAdapters protocolSpecAdapters,
-      final PrivacyParameters privacyParameters,
-      final boolean isRevertReasonEnabled,
-      final EvmConfiguration evmConfiguration,
-      final MiningParameters miningParameters,
-      final BadBlockManager badBlockManager,
-      final boolean isParallelTxProcessingEnabled,
-      final MetricsSystem metricsSystem) {
-    this(
-        config,
-        Optional.empty(),
-        protocolSpecAdapters,
-        privacyParameters,
```

Comment:
if private chain protocol schedules/specs inherited from something other than mainnet, we could remove privacyParameters from mainnet entirely 🤔

Reply:
oh hell yeah. that might be a good aspect to cleave this apart.